packages/cli/tests/test_package/srcWithSyft/syft.ts

```diff
@@ -0,0 +1 @@
+//NOT CHANGED
```

Comment:
hmm. is this an output of a test ?

Reply:
it's just there to support 'init does not create syft file if it already exists'. since everything is testing the file system, this was the most consistent way to support the test case